NOTICE

```diff
@@ -1,5 +1,7 @@
 ros2_rust
 Copyright 2018 Esteve Fernandez (esteve@apache.org)
+Copyright 2019 Easymov Robotics (dev@easymov.fr)
```

Comment:
I don't think it's reasonable for a first contribution to add yourself as an author. I'm more than happy to add you after a few more contributions, that's how opensource works.